.github/workflows/containers.yaml

```
@@ -0,0 +1,48 @@
+name: Container Images Build
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)